app/services/statements/bulk_creator/contract_row.rb

```diff
@@ -0,0 +1,44 @@
+module Statements
+  class BulkCreator
```

Comment:
Historically I've had issues with multiple files defining the same class (which is what this does)

If this file is loaded before `bulk_creator.rb` then you'll end up with an empty instance of `BulkCreator` class defined and it'll never try to autoload the `bulk_creator.rb` file

2 common approaches are either

1) Use a pluralized name as a module - `module BulkCreators`
2) Less pleasant, replace with `BulkCreator.class_eval do` - this will force autoloading the original class, then reopen it and define this `ContractRow` class within it